### ISSUE TITLE
Add Native Animals

### ISSUE THREAD

**Issue by tedbarnett** [TimeWalkOrg/Manhattan#13]
Will be good to have some realistic "fauna" wandering the woods.  Per [this interactive map](http://www.oasisnyc.net/map.aspx) of 1609 Manhattan, these included (in descending order of importance):
- [White Tailed Deer](https://www.nyc.gov/site/wildlifenyc/animals/deer.page)
- Harbor Porpoise (in Hudson and East River only)
- Beaver
- River Otter
- Spotted Sandpiper (shoreline bird)
- Timber Rattlesnakes
- Black Bear (likely rare)
- Gray Wolf (howling at night?)
- Mountain Lion (likely rare)

**Comment by tedbarnett** [comment]
![1609 Habitats](https://github.com/TimeWalkOrg/Manhattan/assets/694013/1cf8e338-ec35-4b0e-ba9f-24115266212b)

**Comment by tedbarnett** [comment]
Should I buy packages for these and we keep them in 3rdPartyAssets?  Like this one: https://www.unrealengine.com/marketplace/en-US/product/animx-forest-animals which I went ahead and bought so I can play with, but we now have it.

**Comment by deedox** [comment]
Okay. they look very good and it's one package with same skeletons and materials.

**Comment by tedbarnett** [comment]
Cool.  Bought it and downloading now.  Thinking I will migrate it into /Content/3DAssets, then share zip file of that folder with you via Google Drive.  Right?

---
***@***.***
https://www.linkedin.com/in/tedbarnett

**Comment by deedox** [comment]
Yes.

**Comment by tedbarnett** [comment]
I tried using it myself to just drop some deer into a field, but it seems designed to make me "into" the deer (i.e. takes control of the UI).  I just want an idling deer.  Not clear how one deploys that.  And maybe a bear wandering in the northern woods.  And some foxes by Minetta Creek.

**Comment by deedox** [comment]
Yeah, maybe some code needs to be adjusted. I have to see for myself. To make them move around we will need some a navigation for the terrain..

**Comment by tedbarnett** [comment]
In 1609, Manhattan was a diverse, vibrant island with a rich variety of flora and fauna. It's worth noting that the exact composition of species is challenging to reconstruct accurately, but there are many historical accounts and scientific studies that give us an understanding of what the environment might have been like.

Flora: The vegetation of Manhattan in 1609 would have been typical of a northeastern temperate forest. It was likely a mixture of deciduous trees like oak, hickory, chestnut, and maple, interspersed with evergreens like pines and hemlocks. The understory would have contained shrubs such as blueberries and huckleberries, and the ground would have been covered with various ferns and wildflowers. Along the rivers and estuaries, there would have been marshlands and wetlands, with plants like cattails, reeds, and various grasses.

Fauna: The fauna of Manhattan in 1609 would have been equally diverse. Deer, beavers, and other small mammals like raccoons, rabbits, and squirrels were probably common. Larger predators such as wolves, mountain lions, and bears might have been present, but less frequent. The waters surrounding the island would have been teeming with fish like sturgeon, herring, and oysters. The skies would have been filled with birds, including waterfowl, raptors, and songbirds.

Please note that this is a general idea based on the types of ecosystems found in the Northeastern United States at that time. The actual species present could have varied. For a more accurate depiction, historical records and archaeological evidence from the period would need to be consulted.
